src/style.css

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700;900&display=swap");
+
+html {
+  font-family: "Roboto", "Arial", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  padding: 30px 14px 14px;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card_image {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0 0 5px;
+}
+
+.card__product-code {
+  font-size: 11px;
+  line-height: 13px;
+  margin-bottom: 15px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__item {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url("./images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__item:nth-child(-n+4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.card__score {
+  align-self: center;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__price-tag {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  margin-bottom: 20px;
+}
+
+.card__price-text {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 170px;
+  height: 40px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  background: #00acdc;
+  color: #fff;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background: #fff;
+  border: 1px solid #00acdc;
```

Comment:
at this moment when you hovered on your link - border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change his position, so its better if you write border property inside the button.

Reply:
Got it. Fixed.